### PR TITLE
Added focus to reset button and corrected heading nesting for search

### PR DESCRIPTION
**Pull request by radinamatic**
## Summary

Search reset button wasn't getting the global outline when in focus (maybe because inside the input field?). I didn't like the default style here anyway, so I tried to add a slight invert filter with a [border for IE support](http://caniuse.com/#feat=css-filters).  

Now I'm thinking that I prefer the `border` property for visual indication of `:focus` state globally :stuck_out_tongue:, but will test that eventually in another PR.

reset-focus

Corrected the nested headings in the search results, decreasing the top margin of `3em` that seemed to leave a lot of free space:

kolibri dev server - mozilla firefox_046
 
IMHO the `$top-offset = 60px` is too much too, especially on smaller devices, and even 40px would suffice, but more design-trained-brain may disagree (@jtamiace @66eli77 @whitzhu), so I didn't change it (yet):

kolibri - chromium_048

### CODE REVIEW

**Comment by indirectlylit** [comment]
I agree that this looks good and realize we aren't 100% consistent at the moment.

However - if possible - I'd like to move towards globally-defined defaults rather than introducing more special cases. Maybe it would be worth looking into where the default is getting over-written? (Someone smarter than me once said: CSS issues are best fixed by deleting code, not adding code.)

> Now I'm thinking that I prefer the border property for visual indication of :focus state globally 😛, but will test that eventually in another PR.

That would be great to see, and another reason to avoid adding a special case here :)

**Comment by rtibbles** [comment]
Does this need updates?

**Comment by radinamatic** [comment]
Yes @rtibbles, I'm weighing options, hope to have the final version today!

**Comment by radinamatic** [comment]
Search reset button wasn't getting the global outline when in focus because of the (most obvious) `outline: none`... :innocent: 

However, once that was removed, focus outline appeared on Chrome but FF displayed only the native focusring:

Chrome
kolibri - chromium_075


FF
kolibri dev server - mozilla firefox_076

After way too much time, and discovering that [FF had this "outline buggy mess" for 14 years](https://bugzilla.mozilla.org/show_bug.cgi?id=140562), the issue may even be [due to Normalize](https://github.com/necolas/normalize.css/issues/481)...?

Anyways, re-stating the outline style in the local .vue file did it for FF too:

kolibri dev server - mozilla firefox_077

File: kolibri/core/assets/src/styles/core-global.styl [hunk 1]
```diff
@@ -99,6 +99,11 @@ button
   &:focus
     outline: $core-action-light 2px solid
 
+  &:disabled
+    color: $core-text-annotation
+    border-color: $core-text-annotation
+    cursor: default
+
```
Comment (radinamatic) [inline kolibri/core/assets/src/styles/core-global.styl:106]: This has nothing to do with the present PR, but after messing with styles for a while, I wanted to roll back on a commit, and decided to grab the core global styles from the latest master.

**Pull request merged**